lib/intrinsicclass.ts

```diff
@@ -1,8 +1,24 @@
-import { Temporal } from '.';
+import bigInt from 'big-integer';
+import { Temporal } from '..';
 
 import { DEBUG } from './debug';
 
-type TemporalIntrinsics = Omit<typeof Temporal, 'Now'>;
+type OmitConstructor<T> = { [P in keyof T as T[P] extends new (...args: any[]) => any ? P : never]: T[P] };
+
+type TemporalIntrinsics = Omit<typeof Temporal, 'Now' | 'Instant' | 'ZonedDateTime'> & {
+  Instant: OmitConstructor<Temporal.Instant> &
```

Comment:
Why is this needed?

Reply:
The constructor in the public types only accepts `bigint`. The implementation constructor accepts `bigInt.BigInteger | bigint`. This hacky type was one way to support this dichotomy. My suggestion would be to not spend a lot of time worrying about it now-- let's solve it "the right way" (whatever that is!) with your JSBI PR.